addon/components/a11y-landmark.js

```diff
@@ -60,5 +87,22 @@ export default Ember.Component.extend({
         }
     }),
 
+    _validateProperties() {
+        this._validateTagName(this.tagName);
+        this._validateLandmarkRole(this.landmarkRole);
+    },
 
-});
\ No newline at end of file
+    _validateTagName(tagName) {
+        if (VALID_TAG_NAMES.indexOf(tagName) === -1) {
+            const validValues = VALID_TAG_NAMES.join(', ');
+            Ember.assert(`Invalid tagName "${tagName}". Must be one of ${validValues}.`);
+        }
+    },
+
+    _validateLandmarkRole(landmarkRole) {
+        if (VALID_LANDMARK_ROLES.indexOf(landmarkRole) === -1) {
+            const validValues = VALID_LANDMARK_ROLES.join(', ');
+            Ember.assert(`Invalid tagName "${landmarkRole}". Must be one of ${validValues}.`);
+        }
+    }
+});
```

Comment:
So this part needs a bit of tweaking. 

For example, if I put this in my template:

`{{#a11y-landmark tagName="header" landmarkRole="contentInfo"}}
    <h1>Header</h1>
{{/a11y-landmark}}
`
This returns the error 
> Invalid tagName "contentInfo". Must be one of banner, navigation, aside, main, form, search, application, document, region.

What it should be doing could be one of two things:
1. if there's a tagName, declared (it means they meant it to be something other than div), ignore the landmarkRole, since it adds it for them. 
2. however, we could still assert that they are wrong (a valid tagName that isn't div was declared, you don't need to assign a landmarkRole) to help users learn how to do this correctly.